share/ansible/roles/ci_run/tasks/opensuse.yml

```diff
@@ -0,0 +1,91 @@
+---
+# tasks file for ci_run
+- name: Ensure python is installed
+  ansible.builtin.raw: zypper --non-interactive in python3
+
+- name: Ensure dependencies are installed
+  community.general.zypper:
+    name:
+      - autoconf
+      - automake
+      - byacc
+      - diffutils
+      - gawk
+      - gcc
+      - gettext-tools
+      - intltool
+      - libcmocka-devel
+      - libtool
+      - libxslt-tools
+      - make
+      - systemd-devel
```

Comment:
Do you really need all these dependencies defined manually? Fedora's spec file already defines things like autoconf, automake, gcc, etc. as dependencies. Thus, we don't need to define them in the ansible playbook.

Reply:
Yeah for some reason I need them. I tried without them at first. Some of those deps we don't use in the spec file, and others seem to be pulled in indirectly. But on that docker image everything seems to be set to install only minimal so I'll have to define them specifically here.

Reply:
Okay, it's not a big deal, so we can go ahead with the current proposal. However, I recommend you to check if those dependencies are also needed in the spec file. Maybe you didn't need them until now because the openSUSE infrastructure already provides an image with them, but if that's not the case then it fails, and thus, it highlights a problem in the spec file.